modules/cert-manager-fips-support.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/index.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="cert-manager-fips-support_{context}"]
+= About FIPS compliance for {cert-manager-operator}
+
+{cert-manager-operator} is designed for FIPS starting v1.14.0 and above. When running on {product-title} in FIPS mode, it uses the RHEL cryptographic libraries submitted to NIST for FIPS Validation on only the x86_64, ppc64le, and s390X architectures. For more information about the NIST validation program, see "cryptographic module validation program". For the latest NIST status for the individual versions of the RHEL cryptographic libraries submitted for validation, see "compliance activities and government standards".
+
+To enable FIPS mode, you must install {cert-manager-operator} on an Openshift Cluster configured to operate in FIPS mode. For more information, see "2.1.4. Do you need extra security for your cluster?”
```

Comment:
The "2.1.4" heading number is not part of the heading and can change w/o notice:
```suggestion
To enable FIPS mode, you must install {cert-manager-operator} on an {product-title} cluster configured to operate in FIPS mode. For more information, see "Do you need extra security for your cluster?"
```